backend/manage.py

```diff
@@ -21,6 +21,12 @@
 def run():
     app.run("0.0.0.0")
 
+@manager.command
```

Comment:
how is this command used?

Reply:
I just changed the Dockerfile to run `python manage.py deploy`. Should I push that change too?